src/App.js

```diff
@@ -1,98 +0,0 @@
-import React from "react";
-import { StrictMode } from "react";
-import ReactDOM from "react-dom";
-import { useState } from "react";
-import Header from "./Header";
-import Question from "./Question";
-import Suggestions from "./Suggestions";
-
-const App = () => {
-    const [quizStarted, setQuizStarted] = useState(false);
-    const quizQuestions = ["How are you feeling today?*", "Do you prefer indoors or outdoor?*", "Would you like to socialize or plan solo?*"];
-    const quizOptions = [["Relaxed", "Cheery", "Snoozy", "Productive"], ["Outdoors", "Indoors"], ["Socialize", "Solo"]];
-    const images = ["https://images.pexels.com/photos/3807758/pexels-photo-3807758.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260", "https://cdn.ramseysolutions.net/media/article-images/budget-date-night-l.jpg", "https://hips.hearstapps.com/clv.h-cdn.co/assets/18/02/1515470256-levi-guzman-268866.jpg"];
-    const [mood, setMood] = useState("");
-    const [location, setLocation] = useState("");
-    const [socialize, setSocialize] = useState("");
```

Comment:
I see that you are tracking this information per question. Though you current approach will not persist this information for every question.

Reply:
hmm reading through this file, I see that each question is tackling a specific data point. Instead of managing the  question and answer separately per question, we can use an array to manage this information in colocated manner